CHANGES.txt

```diff
@@ -36,6 +36,8 @@ Changes
 Fixes
 =====
 
+ - Fixed an issue that causes a lost alias on subselect queries.
```

Comment:
.. that causes aliases used in select list to be lost  if query includes subselect(s).